Readme.md

```diff
@@ -14,9 +14,20 @@ This Container allows users to send E-Mails via SMTP to a Zimbra Web Interface.
 To start the container use the following command
 
 ```
-docker run -p 587:587 ghcr.io/cirosec-studis/zimbraweb-smtp-bridge
+docker run -p 587:587 ghcr.io/cirosec-studis/zimbraweb-smtp-bridge:nightly
```

Comment:
is the nightly tag necessary? Shouldnt it be the same as `latest`?

Reply:
No, I don't think so. `latest` only builds once every time a *new* tag is created. *nightly* builds every night, even if no new tag was created.